### PR TITLE
Add Transactions instructions, clean up getting started page

### LINKED ISSUE #638: Sending a transfer
We should have a section instructing users how to send a transfer using Polkadot-JS Apps.

### PR DESCRIPTION
**Pull request by krichard410**
Closes #638 and I spruced up the Getting Started page to include some more details for brand-new beginners. 
Those additional changes were:
- Added the Get Started button to the navigation, similarly with Kusama Guide
- Mentioned the Mooc and the Teched Explainer videos in the Get Started page (also tried dividing up the content, so specifically beginner content for new blockchain learners, new Polkadot learners, and new Kusama learners)
- I changed the name from Guide to Getting Started, as it might make more sense for new users

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -19,7 +30,14 @@ functionality.
 - [Voting for Councillors][council voting]
 - [Using the Treasury][treasury]
 
+For brand-new learners of Kusama: To learn more about how to build and maintain on the Kusama
+network, please head over to our [Kusama Guide][kusama guide].
+
+[mooc]: https://mooc.web3.foundation/course/blockchain-fundamentals/
+[teched videos]: https://www.youtube.com/watch?v=mNStMPZjiHM&list=PLOyWqupZ-WGuAuS00rK-pebTMAOxW41W8
+[ui]: https://polkadot.js.org/apps/
 [account generation]: https://example.com
```
Comment (lsaether) [inline docs/getting-started.md]:
```suggestion
[account generation]: learn-account-generation
```

File: docs/getting-started.md [hunk 2]
```diff
@@ -1,14 +1,25 @@
...
+For brand-new learners of Blockchain technology:
+
+- The [Blockchain Fundamentals MOOC course][mooc] is a great introduction to familiarizing yourself
+  with crytography, blockchain concepts, decentralization, networks, crytocurrency, and so much
+  more. This is recommended for users with backgrounds of all levels, and is also a free course.
+
+For brand-new learners of Polkadot:
+
+- [Watching the Technical Explainer Videos][teched videos]: These are great introductory videos that
+  explain and demonstrate how to use Polkadot and it's [User Interface][ui].
```
Comment (lsaether) [inline docs/getting-started.md]:
```suggestion
  explain and demonstrate how to use Polkadot and its [User Interface][ui].
```

File: docs/learn-transactions.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
+---
+id: learn-transactions
+title: Transactions
+sidebar_label: Transactions
+---
+
+Transactions are used to send a balance to another user. To begin sending transactions, we will
+learn how to using Polkadot User Interface.
+
+## Polkadot UI
```
Comment (lsaether) [inline docs/learn-transactions.md:10]: I usually call it Polkadot-JS Apps as its proper name, although Polkadot UI seems friendlier.

File: docs/learn-transactions.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+---
+id: learn-transactions
+title: Transactions
+sidebar_label: Transactions
+---
+
+Transactions are used to send a balance to another user. To begin sending transactions, we will
+learn how to using Polkadot User Interface.
```
Comment (Swader) [inline docs/learn-transactions.md]: "learn how to using" sounds weird to me. Maybe, change sentence to "We will learn how to begin sending transactions by using the Polkadot User Interface"

**Review by Swader — APPROVED** [review]
Just a nitpick from me

**Comment by Swader** [comment]
Looks like a link failure snuck in too 🤔

**Comment by krichard410** [comment]
Just realized this PR (specifically the getting started page) could fix #15 as well

**Pull request merged**